metal3/AIR-148_pause-containers-selection.md

```diff
@@ -28,15 +26,17 @@ kubelet --bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf \
 ```
 
 After kubeadm init runs, the values is set in the following file.
-```
-root@kind-control-plane:~# cat /var/lib/kubelet/kubeadm-flags.env 
 
-KUBELET_KUBEADM_ARGS="--cgroup-driver=cgroupfs 
---network-plugin=cni 
+```bash
+root@kind-control-plane:~# cat /var/lib/kubelet/kubeadm-flags.env
```

Comment:
That is a bash prompt if any :) It is generally a bad practice to include prompts, but here I think it conveys information as well (being root, and specific node name), so I left it.

Reply:
Yes I'm fine with it like this also, just making a note that if one wants to include the prompt and also output of commands, then `console` makes more sense than `bash`. Syntax highlighting becomes quite weird with a prompt that ends in `#` when highlighting as `bash` since it interprets the command as a comment :joy: